app.rb

```diff
@@ -0,0 +1,40 @@
+require 'sinatra/base'
+require 'pg'
+require_relative './lib/peep'
+require_relative './lib/user'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  get '/' do
+    @peeps = Peep.all
+    @welcome_message = session[:name] ? "Welcome #{session[:name]}!" : ""
```

Comment:
Nice use of sessions to bring in user's name and differentiate between a non-user and user 👍